scripts/compile-email.js

```diff
@@ -66,12 +66,12 @@ data['collective.expense.paid'] = {
 data['collective.expense.missing.receipt'] = {
   host: { id: 1, name: 'WWCode', slug: 'wwcode' },
   expense: {
-    description: 'Team dinner',
```

Comment:
Why remove this?

Reply:
@Betree : Because we are not using the `description` anymore in the email template.